src/commons/locations.json

```diff
@@ -0,0 +1,18 @@
+{
+    "Seattle": "Seattle",
+    "San_Francisco": "San Francisco / Bay",
+    "Los_angeles": "Los Angeles",
+    "Silicon_Valley": "Silicon Valley",
+    "Chicago": "Chicago",
+    "Boston": "Boston",
+    "New_York": "New York",
+    "Austin": "Austin",
+    "Vancouver": "Vancouver",
+    "Kitchener_Waterloo": "Kitchener-Waterloo",
+    "Toronto": "Toronto",
+    "Peel Region": "Peel_Region",
```

Comment:
I think we shouldn't have spaces for the LHS since those are our variable names to reference.